playwright.config.ts

```diff
@@ -11,6 +11,9 @@ dotenv.populate(
 	})
 );
 
+// We disable the exchange in the E2E tests to load the buttons in the Hero
+process.env.VITE_EXCHANGE_DISABLED = 'true';
```

Comment:
You cannot solve that with a particular `.env` file for the e2e test?

Reply:
> Github Secrets that is for the E2E CI

Not sure, why?

I was thinking something like

```
# .env.e2e
VITE_EXCHANGE_DISABLED = 'true'
```

and

```
# playwright.config.ts
import { config } from 'dotenv';

config({ path: `.env.e2e` });
```

Not sure if it works.I would just avoid having business flag defined in `playwright.config.ts`.

Reply:
ah! you mean to add the `.env.e2e` file to the repo, like `.env.template` or `.env.example` ? yes, that would work

Reply:
Yes

Reply:
done!